src/content/app/genome-browser/components/track-panel/components/track-panel-list/TrackPanelList.tsx

```diff
@@ -75,7 +75,7 @@ export const TrackPanelList = () => {
       {activeFocusObject?.type === 'gene' &&
       activeGenomeId &&
       activeFocusObject.stable_id ? (
-        <section className="mainTrackItem">
+        <section className={`${styles.mainTrackItem} mainTrackItem`}>
```

Comment:
What's happening here?

If the `mainTrackItem` class that is in the deleted line was used in a test, then you no longer need it (`styles.mainTrackItem` will evaluate to `mainTrackItem` in the test). If it was used for some other purpose, then what was it?

Reply:
It was just for the test. I didnt know that it would evaluate the same in test. Will update.

Reply:
The change you made should have been the opposite. Not this:

<img width="649" alt="image" src="https://user-images.githubusercontent.com/6834224/178254917-3f3317bb-1b47-4b88-8fed-de396d16889f.png">

but this:

` <section className={`${styles.mainTrackItem}`}>`